### PR TITLE
IMU Transformation AL article & Small IMU Docs Update

### PR DESCRIPTION
**Pull request by marc-tonsen**

### CODE REVIEW

File: neon/data-collection/data-streams/index.md [hunk 1]
```diff
@@ -84,20 +84,22 @@ Audio recording is disabled in the Neon Companion app by default and can be enab
 <Badge>Real-time</Badge><Badge>Pupil Cloud</Badge><Badge>Neon Player</Badge>
 The Neon module is equipped with a 9-DoF [inertial measurement unit](https://invensense.tdk.com/products/motion-tracking/9-axis/icm-20948/) (IMU) featuring an accelerometer, gyroscope, and magnetometer. The accelerometer and gyroscope measure linear acceleration and angular velocity, respectively, and are provided as raw values.
 
-A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation relative to magnetic north and gravity as a quaternion. Note that in order to obtain precise absolute yaw readings, the magnetometer needs to be [calibrated](/data-collection/calibrating-the-imu/).
+A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation relative to magnetic north (positive world y-axis), gravity (negative world z-axis), and a rightward pointing vector (positive world x-axis) as a quaternion. We refer to this as the world coordinate system. It is important to note that this is not the same as the local IMU coordinate system.
```
Comment (marc-tonsen) [inline neon/data-collection/data-streams/index.md:87]:
```suggestion
A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation in the world as a quaternion. The world coordinate system is defined such that magnetic north corresponds to the positive y-axis, gravity corresponds to the negative z-axis, and a rightward pointing vector to these corresponds to the positive x-axis.
```
I propose this slightly altered version. IMO a formulation that says "The world coordinate system is [...]" is easier to understand.

File: neon/data-collection/data-streams/index.md [hunk 2]
```diff
@@ -84,20 +84,22 @@ Audio recording is disabled in the Neon Companion app by default and can be enab
...
+![IMU Scene Camera](./imu-scene_camera_offset-black.png)
 
 ### Euler Angles
 
-When exporting recordings from Pupil Cloud or Neon Player the IMU's orientation in Euler angles (i.e. roll, pitch, and yaw) is also available.
+When exporting recordings from Pupil Cloud or Neon Player the IMU's orientation in Euler angles (i.e. pitch, yaw, and roll) is also available:
 
-Pitch is defined as a rotation around the x-axis with a value range of -90° to +90°. Yaw and roll are rotations around the y- and z-axis, respectively, with value ranges of -180° to +180°.
+- Pitch is defined as a rotation around the world x-axis with a value range of -90° to +90°. Wearing Neon upright and looking parallel to the horizon roughly corresponds to 0° pitch. Backward tilt is positive, forward is negative.
+- Yaw is a rotation around the world z-axis with a value range of -180° to +180°. With a calibrated IMU, a yaw of 0° indicates alignment with magnetic north. Leftward turn is positive, rightward is negative.
```
Comment (marc-tonsen) [inline neon/data-collection/data-streams/index.md:102]:
```suggestion
- Yaw is a rotation around the world z-axis with a value range of -180° to +180°. With a [calibrated IMU](/data-collection/calibrating-the-imu/), a yaw of 0° indicates alignment with magnetic north. Leftward turn is positive, rightward is negative.
```

File: neon/data-collection/data-streams/index.md (hunk 1 above)
Comment (rennis250) [reply neon/data-collection/data-streams/index.md:87]: Yes, thanks!

**Comment by mikelgg93** [comment]
@rennis250 Sy, for hijacking this PR a bit, there was a typo, and since this PR is gonna be soon merged, I thought it would be easier to modify it here rather than merging and potentially solve conflicts.

**Comment by rennis250** [comment]
@mikelgg93 np, thanks!

**Comment by mikelgg93** [comment]
Vimeo might be down and the timeouts are fine, but those 2x 404 errors need fixing. Do you want me to fix them @rennis250 ?

## Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 6188  |
| ✅ Successful | 4286  |
| ⏳ Timeouts   | 1     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 1894  |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 4     |

## Errors per input

### Errors in docs_merged/alpha-lab/imu-transformations/index.html

* [404] [http://docs.pupil-labs.com/alpha-lab/imu-transformations/#application-example](http://docs.pupil-labs.com/alpha-lab/imu-transformations/#application-example) | Failed: Network error: Not Found

### Errors in docs_merged/core/diy/index.html

* [401] [https://player.vimeo.com/video/59844058](https://player.vimeo.com/video/59844058) | Failed: Network error: Unauthorized

### Errors in docs_merged/neon/data-collection/data-streams/index.html

* [404] [https://docs.pupil-labs.com/alpha-lab/imu-transformations/](https://docs.pupil-labs.com/alpha-lab/imu-transformations/) | Failed: Network error: Not Found

### Errors in docs_merged/core/academic-citation/index.html

* [ERR] [http://dl.acm.org/citation.cfm?doid=2638728.2641695](http://dl.acm.org/citation.cfm?doid=2638728.2641695) | Failed: Too many redirects: error following redirect for url (https://dl.acm.org/doi/10.1145/2638728.2641695?cookieSet=1): too many redirects
* [TIMEOUT] [http://hdl.handle.net/1721.1/72626](http://hdl.handle.net/1721.1/72626) | Timeout

**Pull request merged**